dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jca.resourcedefinition.jms-2.0.feature

```diff
@@ -10,3 +10,4 @@ IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature
 IBM-Install-Policy: when-satisfied
 kind=ga
 edition=base
+WLP-Activation-Type: parallel
```

Comment:
Do you want to do io.openliberty.connectors.resourcedefinition.messaging-3.0.feature as well?

Reply:
@dazavala is working on the messaging-3.0 feature.  I'll leave the messaging features out of this update for now.

Reply:
Nevermind, I thought was was referring to the public feature.  Updated :)